diskdict/diskdict.py

```diff
@@ -0,0 +1,91 @@
+import os
+import shelve
+
+from deeputil import Dummy
+
+DUMMY_LOG = Dummy()
+
+# export DISKDICT_DATAFILE_PATH=='/home/usr/diskdict_fiile_path'
```

Comment:
@prasannababuAddagiri Do you need this comment?

Reply:
For explaining to user purpose only